msbuild/Xamarin.Mac.Tasks.Core/Tasks/MmpTaskBase.cs

```diff
@@ -72,6 +72,10 @@ public class MmpTaskBase : ToolTask
 		public string I18n { get; set; }
 		public string ExtraArguments { get; set; }
 
+		public string AOTMode { get; set; }
+		public bool HybridAOT { get; set; }
```

Comment:
AOTScope maybe (based on sebastians comment below)?

Related: Should it just be AotScope, etc for the names instead?  Similar to how we rename NSURL names to NSUrl